src/__tests__/types.ts

```diff
@@ -92,3 +93,10 @@ assert<Has<typeof resArrayDefault, undefined>>(false);
 const resMaybeNullWithDefaultNull = oc(x).a.maybeNull(null);
 assert<Has<typeof resMaybeNullWithDefaultNull, string>>(true);
 assert<Has<typeof resMaybeNullWithDefaultNull, null>>(true);
+
+// Can traverse any sub-structures
+const resAnyValue = oc(x).a.any();
+assert<IsAny<typeof resAnyValue>>(true);
+
+const resAnyDeepValue = oc(x).a.any.moo.says.the.cow('with default');
+assert<IsAny<typeof resAnyDeepValue>>(true);
```

Comment:
Shouldn't it be string?

Reply:
The default here just so happens to be of type `string`...but the underlying type is still `any` per our definition of `interface X` in this test case.

Reply:
Ah right! You cannot be of course ever sure what it returns.